boards: Added new board Raytac mdbt50q-db-40

RaytacStanley opened the pull request:
Added new board Raytac mdbt50q-db-40.

RaytacStanley committed:
boards: Added new board Raytac mdbt50q-db-40

Added new board Raytac mdbt50q-db-40.

RaytacStanley committed:
boards: Added new board Raytac mdbt50q-db-40

Added new board Raytac mdbt50q-db-40.

RaytacStanley committed:
Update test_adc.c

RaytacStanley committed:
Update test_adc.c

nordicjm commented:
@RaytacStanley Can you follow the guidelines here for your commits https://docs.zephyrproject.org/latest/contribute/guidelines.html#commit-guidelines
i.e. "Update test_adc.c" is not a valid commit title and empty commit message is also not valid

nordicjm commented on boards/arm/raytac_mdbt50q_db_40_nrf52840/board.cmake:
```diff
@@ -0,0 +1,8 @@
+include(${ZEPHYR_BASE}/boards/common/nrfjprog.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/pyocd.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/openocd-nrf5.board.cmake)
```
328KB is far too large for an image, please vastly reduce width and height of the image and then use https://imagecompressor.com/ to shrink the size down

RaytacStanley replied:
Hi Jamie,
Thank you for your reminder. I will correct the problem you mentioned and upload it again.

RaytacStanley committed:
boards: Fix the problem that the picture is too large

Fix the problem that the MDBT50Q-DB-40 image is too large, and the size is reduced from 320KB to 50KB.

RaytacStanley committed:
boards: Fix the problem that the picture is too large

Fix the problem that the MDBT50Q-DB-40 image is too large, and the size is reduced from 320KB to 50KB.

RaytacStanley committed:
Merge branch 'zephyr-mdbt50q-40' of https://github.com/RaytacStanley/zephyr into zephyr-mdbt50q-40
